src/Query/Builder.php

```diff
@@ -55,6 +56,33 @@ public function updateOrInsert(array $attributes, array $values = [])
         return (bool) $this->take(1)->update(Arr::except($values, array_keys($attributes)));
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function upsert(array $values, $uniqueBy = null, $update = null)
```

Comment:
`$uniqueBy` of `parant:upsert()` is `array|string`. It does not seem to be null.

https://github.com/laravel/framework/blob/11.x/src/Illuminate/Database/Query/Builder.php#L3674

Reply:
```suggestion
    public function upsert(array $values, $uniqueBy = [], $update = null)
```